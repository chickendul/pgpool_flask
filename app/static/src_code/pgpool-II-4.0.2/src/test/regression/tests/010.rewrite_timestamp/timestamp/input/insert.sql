version https://git-lfs.github.com/spec/v1
oid sha256:876be8e63e5402dec4205c0f0ff7c6a783d621250fc5aa716be895481a93f74d
size 791
