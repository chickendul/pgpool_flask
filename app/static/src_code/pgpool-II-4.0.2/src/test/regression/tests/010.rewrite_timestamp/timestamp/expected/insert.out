version https://git-lfs.github.com/spec/v1
oid sha256:39e50c44a46fc269f8f136fa2ac810a1e4ae372dd53fbbd3861c878c4e824d40
size 1740
